docs/en/observability/ci-cd-observability.asciidoc

```diff
@@ -427,3 +427,42 @@ The Concourse CI doesn’t report health metrics through OpenTelemetry. However,
 https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanmetricsprocessor#span-metrics-processor[OpenTelemetry Collector Span Metrics Processor]
 to derive pipeline execution traces into KPI metrics like throughput and the error rate
 of pipelines.
+
+
+[discrete]
+[[check-service-health-from-cd-pipelines]]
+== Check service health from deployment pipelines
+
+
+Integrating automated service health checks in deployment pipelines is critical to end-to-end deployment automation which is critical to increase the deployment frequency .
+
+Elastic Observability exposes service HTTP health check APIs that that integrate in deployment pipelines to verify the behaviour of newly deployed instances
+and automatically continue the deployments or rollback according to the health statuses.
+
+
+image::images/ci-cd-canary-deployment-pipeline.png[Canary Deployment Pipeline]
+
+
+The primary HTTP API to check the health of a service is compare the transaction error rate with a threshold value.
```

Comment:
@kuisathaverat Can we please describe the URL pattern with a description of each parameter?

Reply:
let me know if it is enough